data/events.json

```diff
@@ -148,6 +148,24 @@
     ]
   },
   {
+    "title": "Java User Group Mannheim",
```

Comment:
Please include the title of your talk and not just the name of the event/conference.

Do something similar to the other entries that use a hyphen to separate the two, and please do that for all of talks.

Cheers